src/main/java/com/example/animalshelter/service/AnimalApiError.java

```diff
@@ -0,0 +1,19 @@
+package com.example.animalshelter.service;
+
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+
+@Data
+public class AnimalApiError {
+    private HttpStatus status;
+    private String message;
```

Comment:
Please make these fields as a `final` and remove constructors - `@Data` contains [@RequiredArgsConstructor](https://projectlombok.org/features/constructor) so we shouldn't need explicit constructors here.

Reply:
Looks good. Thanks! :+1:

Reply:
Ok, thanks.